types/brotli/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for brotli 1.3
+// Project: https://github.com/foliojs/brotli.js
+// Definitions by: Artur Androsovych <https://github.com/arturovt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
```suggestion
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
/// <reference types="node" />
```
will be used accross DT package, simpler to maintain.